rust/vectordb/src/table.rs

```diff
@@ -396,17 +443,8 @@ impl NativeTable {
         self.dataset.lock().expect("lock poison").version().version
     }
 
-    /// Create a scalar index on the table
-    pub async fn create_scalar_index(&self, column: &str, replace: bool) -> Result<()> {
-        let mut dataset = self.clone_inner_dataset();
-        let params = ScalarIndexParams::default();
-        dataset
-            .create_index(&[column], IndexType::Scalar, None, &params, replace)
-            .await?;
-        Ok(())
-    }
-
-    pub async fn optimize_indices(&mut self, options: &OptimizeOptions) -> Result<()> {
+    async fn optimize_indices(&self, options: &OptimizeOptions) -> Result<()> {
+        info!("LanceDB: optimizing indices: {:?}", options);
```

Comment:
this should be debug level?

Reply:
I was meant to make this show progress of optimization tasks, as it is a slow process.

I am fine either way.